test/RelayServer.test.ts

```diff
@@ -751,8 +749,8 @@ contract('RelayServer', function (accounts) {
     beforeEach(function () {
       origWorker = relayServer._worker
       started = false
-      // @ts-ignore
-      relayServer._worker = function () {
+      relayServer._worker = async function () {
+        await Promise.resolve()
```

Comment:
what does `await Promise.resolve()` does here?

Reply:
Only prevents linter from complaining that it's async. But it has to be async, otherwise `.then` get thrown in the server.